javaslang/src-gen/main/java/javaslang/Tuple8.java

```diff
@@ -571,17 +571,13 @@ public String toString() {
     }
 
     @Override
-    public <T> Tuple prepend(T value) {
-        throw new UnsupportedOperationException("Prepend to Tuple8");
-    }
-
-    @Override
     public <T> Tuple append(T value) {
         throw new UnsupportedOperationException("Append to Tuple8");
     }
 
-    public static <T1, T2, T3, T4, T5, T6, T7, T8> Tuple8<Seq<? extends T1>, Seq<? extends T2>, Seq<? extends T3>, Seq<? extends T4>, Seq<? extends T5>, Seq<? extends T6>, Seq<? extends T7>, Seq<? extends T8>> sequence(Iterable<Tuple8<? extends T1, ? extends T2, ? extends T3, ? extends T4, ? extends T5, ? extends T6, ? extends T7, ? extends T8>> tuples) {
-      Objects.requireNonNull(tuples, "tuples is null");
-      return new Tuple8<>(Iterator.ofAll(tuples).map(Tuple8::_1).toList(), Iterator.ofAll(tuples).map(Tuple8::_2).toList(), Iterator.ofAll(tuples).map(Tuple8::_3).toList(), Iterator.ofAll(tuples).map(Tuple8::_4).toList(), Iterator.ofAll(tuples).map(Tuple8::_5).toList(), Iterator.ofAll(tuples).map(Tuple8::_6).toList(), Iterator.ofAll(tuples).map(Tuple8::_7).toList(), Iterator.ofAll(tuples).map(Tuple8::_8).toList());
+    @Override
+    public <T> Tuple prepend(T value) {
+        throw new UnsupportedOperationException("Prepend to Tuple8");
```

Comment:
* you're modifying the code you have previously modified in the same PR. Please rebase it, it confuses reviewers.
* I still don't think it's a good idea to throw here :/